apps/docs/pages/guides/platform/branching.mdx

```diff
@@ -44,35 +47,36 @@ Each Supabase project can consist of multiple Preview Branches. Each Preview Bra
 
-By creating new SQL migrations in a Preview Branch to propose schema changes, those changes can then be merged into your Production branch, which then runs the new migrations on your Supabase Production branch.
+By creating new SQL migrations in a Preview Branch to propose schema changes, those changes can then be merged into your Production branch. These changes then run the new migrations on your Supabase Production branch. This is exactly how Git branches work: experiment on a feature branch and merge back into Production when you're finished.
 
-<Admonition type="note">
+<Admonition type="note" label="Schema-only changes">
 
-Only schema changes are captured in preview branches.
+Only *schema* changes are captured in preview branches.
 
-Preview instances contain no data by default. You can include a seed file to seed your preview instance with sample data when the Preview Branch is created. Future versions of branching may allow for automated data seeding and cloning.
+Preview Branch instances contain no data by default. You must include a seed file to seed your preview instance with sample data when the Preview Branch is created. Future versions of Branching may allow for automated data seeding and cloning.
 
 </Admonition>
 
 ## Git providers
 
 To facilitate Supabase Branching, your Supabase project will be connected to a Git repository via an integration.
 
-Supabase Branching works with [GitHub](#branching-with-github). Future iterations may support other Git providers, such as GitLab and BitBucket.
+Supabase Branching works with [GitHub](#branching-with-github). Future iterations may support other Git providers. If you are interested in other Git providers or a non-Git-based workflow, join the discussions for [GitLab Branching, BitBucket Branching, and non-Git based Branching](https://github.com/orgs/supabase/discussions/18936).
+
 
 ### Branching with GitHub
 
 Supabase Branching uses the Supabase GitHub integration to read files from your GitHub repository. With this integration, Supabase watches all commits, branches, and pull requests of your GitHub repository.
 
-One of your Git branches is your Production Branch. This should also be your Supabase project's Production Branch.
+In Git, you have a Production Branch (typically this is `main`, `master`, `prod`, etc). This should also be your Supabase project's Production Branch.
 
-For every Git Branch in your GitHub repository, you may have a corresponding Preview Branch. Each time a new Preview Branch is created, the migrations in the Git branch of that Preview Branch are run on the Preview Branch.
+You can create a corresponding Preview Branch for any Gith Branch in your repository. Each time a new Preview Branch is created, the migrations in the Git branch of that Preview Branch are run on the Preview Branch. Just like in Git, a Preview Branch is an exact copy of the Production Branch at the time of creation.
```

Comment:
I don't capitalize branch in 'Git branch' - Only 'Preview Branch' as it's one of our features.

Reply:
```suggestion
You can create a corresponding Preview Branch for any Git branch in your repository. Each time a new Preview Branch is created, the migrations in the Git branch of that Preview Branch are run on the Preview Branch. Just like in Git, a Preview Branch is an exact copy of the Production Branch at the time of creation.
```

> Just like in Git, a Preview Branch is an exact copy of the Production Branch at the time of creation.

Not sure this is true for everyone. It's generally the working practice though for many though.